api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,237 @@
+            <not>
+                <foreignKeyConstraintExists foreignKeyName="repeat_defines_attribute_type" />
+            </not>
+        </preConditions>
+        <addForeignKeyConstraint baseColumnNames="person_attribute_type_id" baseTableName="xforms_person_repeat_attribute" constraintName="repeat_defines_attribute_type" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="person_attribute_type_id" referencedTableName="person_attribute_type" />
+    </changeSet>
+    <changeSet author="aman (generated)" id="xforms-1598359669281-1100">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <foreignKeyConstraintExists foreignKeyName="repeat_identifies_person" />
+            </not>
+        </preConditions>
+        <addForeignKeyConstraint baseColumnNames="person_id" baseTableName="xforms_person_repeat_attribute" constraintName="repeat_identifies_person" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="person_id" referencedTableName="person" />
+    </changeSet>
+    <changeSet author="aman (generated)" id="xforms-1598359669281-1172">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <foreignKeyConstraintExists foreignKeyName="user_who_created_xform" />
+            </not>
+        </preConditions>
+        <addForeignKeyConstraint baseColumnNames="creator" baseTableName="xforms_xform" constraintName="user_who_created_xform" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="user_id" referencedTableName="users" />
+    </changeSet>
+    <changeSet author="aman (generated)" id="xforms-1598359669281-1175">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <foreignKeyConstraintExists foreignKeyName="user_who_last_changed_xform" />
+            </not>
+        </preConditions>
+        <addForeignKeyConstraint baseColumnNames="changed_by" baseTableName="xforms_xform" constraintName="user_who_last_changed_xform" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="user_id" referencedTableName="users" />
+    </changeSet>
+    <changeSet author="aman" id="xforms-25082020">
+    	<comment> Adding uuids to all data in the the xform table </comment>
+		<sql>update xforms_xform set uuid = uuid() where uuid is null;</sql>
+    </changeSet>
+    <changeSet author="aman" id="xforms_update_columnname-31082020">
+        <preConditions onFail="MARK_RAN">
+    		<columnExists tableName="xforms_medical_history_field" columnName="tabIndex"></columnExists>
+   		</preConditions>
+   		<comment> Rename tabIndex to tab_index for table xforms_medical_history_field, Present since 4.3.13 </comment>
+	    <renameColumn columnDataType="INT" newColumnName="tab_index" oldColumnName="tabIndex" tableName="xforms_medical_history_field"/> 
```

Comment:
@dkayiwa , for instances that already have column "tabIndex" , I have added this changeset to rename the column to "tab_index" . So, now there should not be a problem if one updates to this version of xform.